pypeit/core/flexure.py

```diff
@@ -169,10 +177,27 @@ def spec_flex_shift(obj_skyspec, sky_file, spec_fwhm_pix=None, mxshft=20, excess
 
     msgs.warn("If we use Paranal, cut down on wavelength early on")
 
-    # Load Archival sky spectrum
-    arx_skyspec, arx_fwhm_pix = get_archive_spectrum(sky_file, obj_skyspec=obj_skyspec, spec_fwhm_pix=spec_fwhm_pix)
+    if sky_file is None and arx_skyspec is None:
```

Comment:
Perhaps add a comment along the lines of `# Check input mode` or something like that?

Reply:
Done